modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements_controller.rb

```diff
@@ -27,7 +27,7 @@ class AppealsApi::V1::DecisionReviews::NoticeOfDisagreementsController < Appeals
 
   def create
     @notice_of_disagreement.save
-    AppealsApi::PdfSubmitJob.perform_async(@notice_of_disagreement.id, 'AppealsApi::NoticeOfDisagreement', api_version)
+    AppealsApi::PdfSubmitJob.perform_async(@notice_of_disagreement.id, 'AppealsApi::NoticeOfDisagreement')
```

Comment:
I think being explicit about the api version argument is better in this case, even if it's just to pass `v1` as a string into it here.

Reply:
The pdf submit job defaults to 'v1'

Reply:
It does, but I still think being explicit on the version we're using makes things a little more clear.